broker_settings.yaml.example

```
@@ -4,6 +4,7 @@ inventory_file: "inventory.yaml"
 # Host Settings
 host_username: "root"
 host_password: "<password>"
+host_port: "<port>"
```

this should be host_ssh_port, correct?
```suggestion
host_ssh_port: "<port>"
```